types/elsys/ems-door/reed.schema.json

```diff
@@ -15,7 +15,26 @@
       "title": "Pulse count absolute",
       "description": "Pulse count absolute",
       "type": "number"
+    },
+    "relativePulse1": {
+      "title": "Increment pulse 1 count",
+      "description": "Increment pulse 1 count",
+      "type": "number"
+    },
+    "pulseAbs2": {
```

Comment:
align the naming, either pulseRel1 or absolutePulse2

Reply:
This would be breaking with current datakeys